src/com/company/Main/Main.java

```diff
@@ -0,0 +1,294 @@
+                tram.setType("Tram");
+                System.out.println(tram.toString());
+                System.exit(0);
+            case 0:
+                break;
+
+        }
+        System.out.println("Choose your color - \n" +
+                "    WHITE 9000$,\n" +
+                "    BLUE 11000$,\n" +
+                "    BLACK 15000$,\n" +
+                "    RED 10000$");
+
+        Scanner Colour = new Scanner(System.in);
+        //Colours[] allColours = Colours.values();
+        String colour = Colour.nextLine();
+            switch (Colours.valueOf(colour.toUpperCase())) {
+
+                case WHITE:
+                    trolleybus.setColor(Colours.WHITE);
+                    trolleybus.setPrice(9000);
+                    System.out.println("Your choice is: "+ Colours.WHITE);
+                    break;
+
+                case BLUE:
+                    trolleybus.setColor(Colours.BLUE);
+                    trolleybus.setPrice(11000);
+                    System.out.println("Your choice is: "+ Colours.BLACK);
+                    break;
+
+                case BLACK:
+                    trolleybus.setColor(Colours.BLACK);
+                    trolleybus.setPrice(15000);
+                    System.out.println("Your choice is: "+ Colours.BLACK);
+                    break;
+
+                case RED:
+                    trolleybus.setColor(Colours.RED);
+                    trolleybus.setPrice(10000);
+                    System.out.println("Your choice is: "+ Colours.RED);
+                    break;
+
+                default:
+                    System.out.println("Something bad happened, try again."); ;
+                    System.exit(0);
+            }
+
+            System.out.println("Choose the fuel tank capacity\n");
+            System.out.println("100l, 150l, 200l \n");
+            System.out.println("There is a coefficient of value increase that depends on fuel tank, for \n" +
+                    "100l is 0, for 150l is 1.2 for 200l is 1.4 \n");
+
+
+            Scanner fuelTank = new Scanner(System.in);
+            int yourFuelTank = fuelTank.nextInt();
+
+            switch (yourFuelTank) {
+
+                case 100:
+                    System.out.println("Your choice is:  " + FuelTank.FUEL_TANK_100l.value + "l" );
+                    trolleybus.setFuelTank(100);
+                    trolleybus.setCoefficientPrice(1);
+                    break;
+
+                case 150:
+                    System.out.println("Your choice is: " + FuelTank.FUEL_TANK_150l.value + "l");
+                    trolleybus.setFuelTank(150);
+                    trolleybus.setCoefficientPrice(1.2);
+                    break;
+
+                case 200:
+                    System.out.println("Your choice is: "+ FuelTank.FUEL_TANK_200l.value + "l");
+                    trolleybus.setFuelTank(200);
+                    trolleybus.setCoefficientPrice(1.4);
+                    break;
+            }
```

Comment:
Why do we need here switch case? Previously you created enum with the constructor. Please use it.